unity-client/Assets/Scripts/MainScripts/DCL/Controllers/QuestsController/QuestsController.cs

```diff
@@ -83,14 +80,12 @@ public void UpdateQuestProgress(QuestModel progressedQuest)
                 pinnedQuests.Remove(progressedQuest.id);
 
             //Alex: Edge case. Progressed quest was not included in the initialization.
-            // We invoke quests events but no sections ones.
+            // We invoke quests events but no sections or QuestCompleted one.
             if (!quests.TryGetValue(progressedQuest.id, out QuestModel oldQuest))
             {
                 quests.Add(progressedQuest.id, progressedQuest);
                 OnQuestProgressed?.Invoke(progressedQuest.id);
 
-                if (progressedQuest.isCompleted)
```

Comment:
so we shouldn't check if it's completed?

Reply:
Oh I just looked at the whole file with Alex's indications and saw where the quest completion eventi s being calle, great!